README.md

```diff
@@ -577,20 +443,234 @@ anything. If a company exits at a very high valuation relative to what was expec
+  that employees work to increase the *future* value of the company.
+  At its best, this philosophy fosters an environment of teamwork and a “rising tides lift
+  all boats” mentality.
+  It also encourages longer-term thinking, which is key for company success.
+  As we’ll [discuss later](#offers-and-negotiations), the amount of equity you’re offered
+  usually reflects both your past or current contribution to the company and your commitment
+  to the company in the future.
+- To reduce the company’s spending on cash compensation:
+  By giving equity, a company can often pay less cash to employees now, with the hope of
+  rewarding them later, and put that money toward other investments or operating expenses.
+  This can be essential in the early stages of a company or at other times where there may
+  not be enough revenue to pay large salaries.
+  It can also help recruit senior employees or executives who would otherwise command
+  extraordinarily high salaries.
+
+### What is equity worth?
+
+The key question to anyone who owns equity is, what is it worth, now or later?
+
+- With public companies, you can usually sell any stock you won outright, so the answer is
+  relatively easy:
+  You look up the current market value of the stock.
+  - 🚧 Mention lockup periods etc.
+- ❗️ With private companies, it’s hard to know the “real value” of stock and stock options.
+  Because the value of private company stock is not determined by public trades,
+  shareholders can only make educated guesses about the likely future value, at a time when
+  they will be able to buy or sell the stock.
+  It’s always worth remembering private company stock is simply a legal agreement that
+  entitles you to something of highly uncertain value, and could well be worthless in the
+  future, or highly valuable, depending on the fate of the company.
+- ☝️ We’ll discuss the notion of [fair market value](#409a-valuations) later, but even if a
+  company assigns a value to the stock for tax and accounting purposes, it doesn’t mean you
+  can expect to sell it for that value!
+- 🄳 The ability to buy and sell stock is called **liquidity**. In startups and many private
+  companies, it is typical to hold the stock until the company is sold or becomes public in
+  an [**IPO**](https://en.wikipedia.org/wiki/Initial_public_offering). A sale or IPO is often
+  called an **exit** or **liquidity event**. Sales, dissolutions, and bankruptcy are all called
+  **liquidations**.
+  - 🚧 Expansion and clarification of “becoming public”?
+
+Finally, note that one other way stock is valuable is that public and some private
+companies can pay [**dividends**](http://www.investopedia.com/terms/d/dividend.asp) (allocations
+of a company’s profit determined by the company’s board) to shareholders, but this is rare
+among startups and companies focused on rapid growth.
+
+- 🚧 Explain more why this is.
+  Define dividends better.
+
+### Can you sell private stock?
```

Comment:
It seems a little misleading to devote this much space to the idea of selling private stock. The first point here might be (like when you Google "Can I feed avocado to my dog?"), "Why would you?" It actually helps to explain the purpose of stock options when you get into why selling stock in a private company is a little silly. I would suggest this section following information on normal IPO exercise options.

Reply:
It's a big question, and matters a lot to some people, so does deserve teh space. Probably the take away is it needs a transition section.